e2e/README.md

```diff
@@ -1,2 +1,17 @@
-In order to run tests locally. need to increase docker daemon memory to 8G. (On mac with docker desktop)
+In order to run end-to-end tests locally:
+
```

Comment:
I think that everything that is a command in here should be a copiable code snippet

Reply:
agree, so things like `make cli` should be in code snippets